clang/lib/CodeGen/CGOpenMPRuntimeGPU.cpp

```diff
@@ -639,14 +639,14 @@ static bool hasNestedSPMDDirective(ASTContext &Ctx,
   return false;
 }
 
-static bool supportsSPMDExecutionMode(ASTContext &Ctx,
+static bool supportsSPMDExecutionMode(CodeGenModule &CGM,
                                       const OMPExecutableDirective &D) {
+  ASTContext &Ctx = CGM.getContext();
```

Comment:
This change is a separate NFC change. Either drop it or commit in a separate patch